identity-service/src/routes/captcha.js

```diff
@@ -1,37 +1,26 @@
 const config = require('../config')
-const { handleResponse, successResponse, errorResponseForbidden, errorResponseBadRequest } = require('../apiHelpers')
+const { handleResponse, successResponse, errorResponseBadRequest, errorResponseForbidden } = require('../apiHelpers')
 const { sequelize } = require('../models')
 
 module.exports = function (app) {
-  app.get('/scores', handleResponse(async (req, res, next) => {
-    if (req.headers['x-captcha-score'] !== config.get('captchaScoreSecret')) {
-      return errorResponseForbidden('Not permissioned to view captcha score.')
+  app.get('/scores', handleResponse(async req => {
+    const handle = req.query.handle
+    if (!handle) {
+      return errorResponseBadRequest('Please provide handle.')
     }
 
-    let response = {}
-    const queryParams = req.query
-
-    // If ids are not passed in, return
-    let userIds
-    if (queryParams && queryParams.id) {
-      userIds = queryParams.id
+    if (req.headers['x-captcha-score'] !== config.get('captchaScoreSecret')) {
+      return errorResponseForbidden('Not permissioned to view captcha scores.')
     }
-    if (!userIds) return errorResponseBadRequest('No ids provided')
 
-    // If not valid ids are passed in, return
-    userIds = userIds.split(',').filter(id => parseInt(id))
-    if (!userIds.length) return errorResponseBadRequest('No valid ids provided')
-
-    const recaptchaEntry = await sequelize.query(
+    const recaptchaEntries = await sequelize.query(
       `select "Users"."blockchainUserId" as "userId", "Users"."walletAddress" as "wallet", "BotScores"."recaptchaScore" as "score", "BotScores"."recaptchaContext" as "context"
       from 
         "Users" inner join "BotScores" on "Users"."walletAddress" = "BotScores"."walletAddress"
       where
-        "Users"."blockchainUserId" in (${userIds})`
+        "Users"."handle" = ${handle}`
```

Comment:
is there any way we can make this request by wallet address? i don't think every user has a handle here unfortunately and i think blockchainUserId even by itself is not great.

otherwise, maybe we do a check here like authmiddleware to recover the handle if it's not set?